docs/version0.30.md

```diff
@@ -41,6 +41,19 @@ To learn more about support for OpenJ9 releases, including OpenJDK levels and pl
 
 The format of classes that are stored in the shared classes cache is changed, which causes the JVM to create a new shared classes cache rather than re-creating or reusing an existing cache. To save space, you can remove all existing shared caches unless they are in use by an earlier release. For more information about the `-Xshareclasses` option, including the `destroy` options that you can use to remove caches, see [`-Xshareclasses`](xshareclasses.md).
 
+### Ignored options captured in java dumps
```

Comment:
Pls add this to the table of contents above.

Reply:
Done.